pkg/crc/segment/segment.go

```diff
@@ -77,12 +85,26 @@ func (c *Client) Upload(ctx context.Context, action string, duration time.Durati
 		return err
 	}
 
-	properties := analytics.NewProperties()
+	return c.segmentClient.Enqueue(analytics.Track{
+		UserId:     userID,
+		Event:      action,
+		Properties: a,
+	})
+}
+
+func baseProperties(source string) analytics.Properties {
+	return analytics.NewProperties().
+		Set("version", version.GetCRCVersion()).
+		Set("source", source)
+}
```

Comment:
From the tray we'll not be sending the tty and remote info, but the duration would have made sense to also be included here, or no?

Reply:
Yes this is the biggest issue I have with this code.
It tracks user actions and doesn't report the status of the tray. I think I will add some code later to push to telemetry when the state of the tray change.